src/ts/lib/keyboard.ts

```diff
@@ -127,30 +132,9 @@ class Keyboard {
 
 		this.pressed.push(key);
 
-		this.shortcut(`${cmd}+\\, ${cmd}+.`, e, (pressed: string) => {
-			e.preventDefault();
-
-			if (pressed.match('.') && this.isFocused) {
-				return;
-			};
-
-			commonStore.isSidebarFixed ? sidebar.toggleOpenClose() : sidebar.toggleExpandCollapse();
-		});
-
-		// Navigation
-		if (!this.isNavigationDisabled) {
-			this.shortcut(isMac ? 'cmd+[' : 'alt+arrowleft', e, () => this.onBack());
-			this.shortcut(isMac ? 'cmd+]' : 'alt+arrowright', e, () => this.onForward());
-
-			if (!UtilCommon.getSelectionRange() && isMac) {
-				this.shortcut(`${cmd}+arrowleft`, e, () => this.onBack());
-				this.shortcut(`${cmd}+arrowright`, e, () => this.onForward());
-			};
-		};
-
-		// Close popups and menus
-		this.shortcut('escape', e, () => {
-			e.preventDefault();
+        // Close popups and menus
+        this.shortcut('escape', e, () => {
+            e.preventDefault();
```

Comment:
Moved escape up to allow to escape the keybinding window dialog when all other keys are disabled.